addons/html_builder/static/src/plugins/border_configurator_option.js

```diff
@@ -20,6 +21,38 @@ export class BorderConfigurator extends BaseOptionComponent {
         this.state = useDomState((editingElement) => ({
             hasBorder: this.hasBorder(editingElement),
         }));
+
+        // Convert border-width/radius styles into --box-border-width/radius
+        // variables if border uses Bootstrap classes
+        if (this.props.withBSClass) {
+            const el = this.env.getEditingElement();
+            if (el.style["border-width"]) {
+                setStyle(el, "--box-border-width", el.style["border-width"], {
```

Comment:
Maybe that changed in the new editor but I doubt it is a good idea for an option `setup` to change the website DOM. Even functionally, this really should be reserved to users **using options**.

What I would expect is the switch to the variable use to be done only when the user enter a new value for the option. So just at the place where you do "set --box-border-width to Xpx", change into "set --box-border-width to Xpx and remove the inline style border-width".